test/plots/piePlotTests.ts

```diff
@@ -187,7 +187,8 @@ describe("Plots", () => {
           { value: 5000 },
           { value: 5000 }];
         let dataset = new Plottable.Dataset(data);
-        piePlot.addDataset(dataset).outerRadius(500);
+        piePlot.addDataset(dataset);
+        piePlot.outerRadius(500);
```

Comment:
This is because addDataset returns `Plottable.Plot` now (notice the removal in the `Plottable.Plots.Pie`). 

Is this an API breaking change? If so the code I should add is.
```typescript
// this is in piePlot.ts
public addDataset(dataset: Dataset) {
  super.addDataset(dataset);
  return this;
}
```